cluster-provision/okd/scripts/provision.sh

```diff
@@ -90,31 +90,136 @@ if [ ! -f "/etc/installer/token" ]; then
     exit 1
 fi
 
-mkdir -p /root/install
-cp /install-config.yaml /root/install/
+export CLUSTER_DIR=/root/install
+INSTALL_CONFIG_FILE=$CLUSTER_DIR/install-config.yaml
+
+# we need to install jq and yq to make yaml changes
+dnf install -y jq
+pip install yq
+
+mkdir -p $CLUSTER_DIR
+
+# print version
+/openshift-install version
+
+if [[ $INSTALLER_TAG =~ ^.*4\.1$ ]]; then
+    cp /install-config.yaml $CLUSTER_DIR/
+
+elif [[ $INSTALLER_TAG =~ ^.*4\.2$ ]]; then
+
+    # modify number of workers
+    yq -y '.compute[].replicas = 2' /install-config.yaml > "$INSTALL_CONFIG_FILE"
```

Comment:
hm I can see you define registries on the install time, does it still will need to have two workers?

Reply:
It showed the same problems we both noticed as before, so I think yes, two nodes are needed. Furthermore the kubevirtci cluster also does not come up on cluster-up with two nodes.

Reply:
hm it probably because of the CPU configuration